src/core/abi/abi-method.ts

```diff
@@ -1,12 +1,24 @@
 import sha512 from 'js-sha512'
-import { ABIType, decodeABIValue, encodeABIValue, getABIType, getABITypeName, parseTupleContent } from './abi-type'
+import { ABIType, ABITypeName, decodeABIValue, encodeABIValue, getABIType, getABITypeName, parseTupleContent } from './abi-type'
 import { ABIValue } from './abi-value'
 import { ARC28Event } from './arc28-event'
 import { Arc56Contract, Arc56Method, StructField } from './arc56-contract'
 import { ABITupleType, decodeTuple, encodeTuple } from './types'
 
-export type ABITransactionType = 'txn' | 'pay' | 'keyreg' | 'acfg' | 'axfer' | 'afrz' | 'appl'
-export type ABIReferenceType = 'account' | 'application' | 'asset'
+export enum ABITransactionType {
+  Any = 'txn',
```

Comment:
```suggestion
  Txn = 'txn',
```
I think we keep as Txn, so it matches the rust side.